core/src/reporter/utils.ts

```diff
@@ -179,9 +179,15 @@ export async function sendTransactionDelegatedFee({
   let response
   try {
     response = (
-      await axios.post(endpoint, {
-        ...transactionData
-      })
+      await axios.post(
+        endpoint,
+        {
+          ...transactionData
+        },
+        {
+          timeout: Number(DELEGATOR_TIMEOUT)
+        }
+      )
     )?.data
     _logger.debug(response)
   } catch (e) {
```

Comment:
We have to detect what kind of error happened in this `catch` block and distinguish between `DelegatorTimeoutError` and other errors (`DelegatorServerIssue`).